### PR TITLE
docs(android,iOS): link to capacitor getting started guide

### PR DESCRIPTION
**Pull request by sean-perkins**
Updates the iOS & Android getting started guides on the Ionic Framework docs to link to the Capacitor documentation. The Capacitor documentation is updated/maintained by the team responsible for tracking these updates. 

Documentation cards have been added to help guide users to the Capacitor docs or link to the legacy Capacitor/Cordova docs (leaving those on the v6 documentation). 

The callout banner would be temporary to help guide existing users looking for the previous guide. 

Preview link: 
- iOS: https://ionic-docs-git-sp-native-develop-guides-ionic1.vercel.app/docs/developing/ios
- Android: https://ionic-docs-git-sp-native-develop-guides-ionic1.vercel.app/docs/developing/android

Internally approved by Web Native & Marketing.

### CODE REVIEW

File: docs/developing/android.md [hunk 1]
```diff
@@ -11,218 +11,28 @@ sidebar_label: Developing for Android
...
 
-This guide covers how to run and debug Ionic apps on Android emulators and devices using [Capacitor](../reference/glossary.md#capacitor) or [Cordova](../reference/glossary.md#cordova). Android apps can be developed on Windows, macOS, and Linux.
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
 
-## Android Studio
+:::info Looking for the legacy Android guide?
 
-[Android Studio](https://developer.android.com/studio/) is the IDE for creating native Android apps. It includes the [Android SDK](../reference/glossary.md#android-sdk), which will need to be configured for use in the command line.
+The Developing for Android guide has officially migrated to the Capacitor docs: https://capacitorjs.com/docs/android. Looking for the legacy docs? [Click here](/docs/v6/developing/android).
```
Comment (thetaPC) [inline docs/developing/android.md:19]: - This link, `https://capacitorjs.com/docs/android`, should be inline to be accessible compliant.
- "Click here" also is too vague to be accessible compliant.

This [article](https://business.scope.org.uk/article/how-to-write-better-link-text-for-accessibility) is a good read to understand why links should be set in a certain way for web accessibility.

File: docs/developing/ios.md [hunk 2]
```diff
@@ -12,173 +12,28 @@ skipIntros: true
...
-This guide covers how to run and debug Ionic apps on iOS simulators and devices using [Capacitor](../reference/glossary.md#capacitor) or [Cordova](../reference/glossary.md#cordova). iOS apps can only be developed on macOS with Xcode installed.
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
 
-There are two workflows for running Ionic apps on iOS:
+:::info Looking for the legacy iOS guide?
 
-- [Running with Xcode](#running-with-xcode)
-- [Running with the Ionic CLI](#running-with-the-ionic-cli)
+The Developing for iOS guide has officially migrated to the Capacitor docs: https://capacitorjs.com/docs/ios. Looking for the legacy docs? [Click here](/docs/v6/developing/ios).
```
Comment (thetaPC) [inline docs/developing/ios.md:20]: - This link, `https://capacitorjs.com/docs/android`, should be inline to be accessible compliant.
- "Click here" also is too vague to be accessible compliant.

This [article](https://business.scope.org.uk/article/how-to-write-better-link-text-for-accessibility) is a good read to understand why links should be set in a certain way for web accessibility.

**Review by thetaPC — CHANGES_REQUESTED** [review]

File: docs/developing/android.md (hunk 1 above)
Comment (sean-perkins) [reply docs/developing/android.md]: Let me know your thoughts on the changes 👍

**Review by thetaPC — CHANGES_REQUESTED** [review]
Maybe something like this. 

"If you need to access the legacy documentation, you can find it Access the legacy developing for iOS guide" is what the screen read would read. It doesn't sound as smooth. I'm also open to "If you need to access the legacy documentation, you can find it _under_ the legacy developing for iOS guide"

**Review by thetaPC — APPROVED** [review]
"under" sounds better in my opinion but "at" works too. I'll leave it up to you if you want to switch to "under".

**Review by thetaPC — APPROVED** [review]
LGTM

**Pull request merged**